templates/admin-page.php

```diff
@@ -60,6 +60,14 @@
 				<?php } ?>
 				</table>
 				<?php } ?>
+
+				<?php if ( $has_experimental_checks ) { ?>
+					<h4><?php esc_attr_e( 'Other Options', 'plugin-check' ); ?></h4>
+					<p>
+						<input type="checkbox" value="include-experimental" id="plugin-check__include-experimental" /> <?php esc_html_e( 'Include Experimental Checks', 'plugin-check' ); ?>
```

Comment:
This should be wrapped in a `<label>` so that you can click on the text. Like you can with the other checkboxes.

Note: the `for` attribute is not needed if the label wraps the input.